src/cluster_legacy.c

```diff
@@ -2926,6 +3048,20 @@ int clusterProcessPacket(clusterLink *link) {
     uint16_t type = ntohs(hdr->type);
     mstime_t now = mstime();
 
+    if (IS_PUBSUB_LIGHT_MESSAGE(type)) {
+        if (!link->node || nodeInHandshake(link->node)) {
+            freeClusterLink(link);
+            serverLog(
+                LL_WARNING,
```

Comment:
```suggestion
                LL_NOTICE,
```

Reply:
The node is misbehaving if this happens. I think it's worth a warning.

Reply:
This is indeed a fuzzy area. I think we really need to capture the guidance in a document or maybe code comment. Here is my stab at it, based on my understanding:

1. DEBUG - useful for interactive debugging; not intended for production
2. VERBOSE - enabled for test passes to aid failure investigation; not intended for production
3. NOTICE - something happened but don't worry, operator, the system still works fine. 
4. WARNING - the server needs attention/help ASAP. please do something, operator

Reply:
Yeah, so if a node is misbehaving so it has to be disconnected, it either means a serious bug (worst case it may get stuck in a reconnect-disconnect loop) or it's a malicious node. The "myself" node works, but the peer node has a serious problem that needs attention ASAP, don't you think?

Reply:
There could be transient failures and the system would recover from them. There isn't much an operator could do or should do in this case than just keeping an eye on it. So maybe no. 4 should be amended/qualified with "actionable" too?

That said, I feel pretty strongly that we are missing a log level or two here and the omission of these levels partly contributes to the confusion. Ideally, I would like to have an ERROR and CRITICAL level too. And then we can log any failures under WARNING, actionable failures under ERROR, and abort right after logging a CRITICAL failure.